pkg/container-collection/k8s.go

```diff
@@ -181,3 +192,66 @@ func (k *K8sClient) ListContainers() (arr []Container, err error) {
 	}
 	return arr, nil
 }
+
+func getSocketPathFromKubelet(clientset *kubernetes.Clientset, nodeName string) (string, error) {
```

Comment:
nit: perhaps be more specific and say it's for getting the container runtime socket path? At the beginning I was thinking this code was getting the socket path to talk to the kubelet. Is `getContainerRuntimeSocketPath` better?